x-pack/plugins/observability_solution/apm/kibana.jsonc

```diff
@@ -29,7 +29,8 @@
       "dataViews",
       "lens",
       "maps",
-      "uiActions"
+      "uiActions",
+      "logsDataAccess"
```

Comment:
**nit**: I don't see any usage of this dependency in the PR, is this a leftover of some removed change?

Reply:
It will be used on a next PR. I can leave it out for now if that matters.